portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ConnectionPropertyList.testcase

```diff
@@ -74,11 +74,13 @@ definition {
 		}
 
 		task ("Cancel to exit the wizard") {
-			ACDXPSettings.cancelWizardConfiguration();
+			ACDXPSettings.goToInstanceSettingsAC();
 		}
 
 		task ("Go to the properties page in the sidebar") {
-			ACDXPSettings.navToSyncedSitesMenu();
+			Click(
+				key_text = "Next",
+				locator1 = "Button#ANY");
```

Comment:
So the aim of this test is to see the pagination _during_ the wizard and then in the sidebar, is that correct? But since we can`t exit the wizard anymore, what do you think about finishing the wizard and then coming back to it and selecting the Property page in the sidebar? This way I think we can assure that we are validating the property pagination during the wizard and out of the wizard.
cc: @ViniLopes20

Reply:
Maybe a similar approach can be applicable to the other tests in this pull request as well? Tell me what you guys think. Thank you!

Reply:
I agree with Manu. I think you could create a new macro that whenever the next/finished button is present then the click would be done and when it doesn't appear then the wizard would be finished. What do you think?

Reply:
Thanks for the suggestion! What do you guys think of changing the cancel wizard to finish the wizard connection instead?

Reply:
Yeah, I think it is good. Thanks Josh

Reply:
Thank you, Josh!